chat-widget/src/components/webchatcontainerstateful/WebChatContainerStateful.tsx

```diff
@@ -203,8 +203,12 @@ export const WebChatContainerStateful = (props: ILiveChatWidgetProps) => {
         .webchat__bubble:not(.webchat__bubble--from-user) .webchat__bubble__content {
             border-radius: 0 !important; /* Override border-radius */
         }
+
+        .webchat__stacked-layout_container>div {
+            background: ${(props?.webChatContainerProps?.containerStyles as IRawStyle)?.background?? ""}
+        }
         `}</style>
```

Comment:
I wonder if this should be a better place to apply the change, it will honor the update of style each time there is a change

 https://github.com/elopezanaya/omnichannel-chat-widget/blob/0fa36a3533abfee2bdd9af7b07ec749de294a8a5/chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx#L82-L83

Reply:
This is not for activity , this class is added for the transcript container background